utils/profile.go

```diff
@@ -29,12 +28,12 @@ func WalkIOSProvProfiles(walkFunc IOSProvProfileWalkFunc) error {
 	}
 
 	for _, pth := range pths {
-		plistData, err := provisioningprofile.NewPlistDataFromFile(pth)
+		profile, err := provisioningprofile.NewPlistDataFromFile(pth)
```

Comment:
shouldn't this still be `plistData`? I mean the function is `NewPlistDataFromFile`